pytest_fixtures/component/rh_cloud.py

```diff
@@ -4,10 +4,10 @@
 
 
 @pytest.fixture(scope='module')
-def rhcloud_manifest_org(module_target_sat, module_extra_rhel_entitlement_manifest):
+def rhcloud_manifest_org(module_target_sat, module_extra_rhel_sca_manifest):
     """A module level fixture to get organization with manifest."""
     org = module_target_sat.api.Organization().create()
-    module_target_sat.upload_manifest(org.id, module_extra_rhel_entitlement_manifest.content)
+    module_target_sat.upload_manifest(org.id, module_extra_rhel_sca_manifest.content)
```

Comment:
`module_extra_rhel_sca_manifest` uses the `extra_rhel_entitlement` manifest category [1], so this change will not reduce the subscription footprint of the rhcloud tests. Please use `module_sca_manifest` with this fixture.

[1] https://github.com/SatelliteQE/robottelo/blob/21b5935ddfeb7f3a8cf36d1acc0cf0a064c97036/pytest_fixtures/component/taxonomy.py#L204